src/OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper/OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper.csproj

```diff
@@ -1,5 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="..\CommonExcludedAssets.props" />
```

Comment:
This project does only logger work and does not require any of the packages in `CommonExcludedAssets.props` as all of those are related to traces. 

We have a project reference to `OpenTelemetry.AutoInstrumentation.csproj` and that indirectly adds assets to the our package even when excluded in parent project. This could be a dotnet bug. 

This line of code could act as a workaround, but its better to leave a comment saying it is not used by this project and only added to avoid assets.

Reply:
What the bootstrapper uses from `OpenTelemetry.AutoInstrumentation.csproj`? Because it is used for the bytecode injection it is preferable that we only load `OpenTelemetry.AutoInstrumentation` when doing bytecode injection. That said is not a hard requirement, but, something that I may want to do while fixing strong name (otherwise we have to strong name a bunch of projects because of the cascading nature of strong name).

Reply:
Bootstrapper uses `LogSettings` from `OpenTelemetry.AutoInstrumentation.csproj`. Other option is to move the configuration read to a shared project.

Reply:
Thanks. No need to change it right now but something to consider carefully.

Reply:
Also since the AspNet module is for .NET core the strong name can be ignored.